### DISCUSSION TITLE
Will there be TypeScript/JavaScript version?

### DISCUSSION THREAD

**Discussion by hodgesz** [stanfordnlp/dspy#755] [Q&A]
We really love what you are doing with DSPy, but our team is more versed with TypeScript or JavaScript vs Python, so we're curious if you might one day release a TypeScript or JavaScript version of the library.

**Comment by hodgesz** [comment]
Any thoughts at all?

**Comment by listenaddress** [comment]
See https://github.com/dosco/llm-client/
